packages/block-library/src/query/edit/inspector-controls/index.js

```diff
@@ -101,6 +100,10 @@ export default function QueryInspectorControls( props ) {
 		return onChangeDebounced.cancel;
 	}, [ querySearch, onChangeDebounced ] );
 	const showInheritControl = isControlAllowed( allowedControls, 'inherit' );
+	const inheritControlLabel = __( 'Contents' );
+	const inheritControlHelp = __(
+		"Block will display items matching the page's context, e.g. category archive or search result"
```

Comment:
```suggestion
		"Block will display items matching the page's context, e.g. category archive or search result."
```

It is a good idea to end all help text with a period.

Reply:
I think "Display a list of posts or custom post types based on the current template." is simple and concise, yet contextual for developers. 

Shared here: https://github.com/WordPress/gutenberg/issues/63598#issuecomment-2241374446

Reply:
We should also use help text for when it's set to custom too, 

"Display a list of posts or custom post types based on specific criteria." 

otherwise there's layout shift, and you don't really know what "custom" does.